internal/exec/stages/disks/filesystems.go

```diff
@@ -190,17 +190,21 @@ func (s stage) createFilesystem(fs types.Filesystem) error {
 		if fs.Label != nil {
 			args = append(args, "-n", *fs.Label)
 		}
+	case "none":
+		// The user specifies this format to skip the creation of a filesystem on a block device.
+		// For more information https://github.com/coreos/ignition/issues/1172
```

Comment:
Minor: meh, not sure if it's worth linking to the originating RFE for this. But WFM too!

Reply:
Oh I meant, I don't think it's worth linking anything at all, not suggesting to link to the Jira issue. :)

Reply:
whoops, didn't read it properly